vars/main.yml

```diff
@@ -1,4 +1,5 @@
 ---
 
 # RKE2 installation method (do not change this option)
-rke2_method: tar
\ No newline at end of file
+rke2_method: tar
+rke2_additional_sans: []
```

Comment:
@cpu-lz the default value of rke2_additional_sans should not be here. Can you move it to defaults/main.yml, plz?